src/error.rs

```diff
@@ -381,6 +381,9 @@ pub struct WriteConcernError {
     /// A description of the error that occurred.
     #[serde(rename = "errmsg")]
     pub message: String,
+
+    #[serde(rename = "errInfo")]
+    pub details: Option<Document>,
```

Comment:
Since this field is public, we should document what its contents might contain. 

From the CRUD spec:

>A document identifying the write concern setting related to the error.

Reply:
not sure how I missed that haha. Thank you! Fixed.